.github/workflows/ci.yaml

```diff
@@ -0,0 +1,29 @@
+on:
```

Comment:
I just noticed that the `kokoro` check is already running these same nox tests so the github action is not needed. See example build log [here](https://source.cloud.google.com/results/invocations/c2a84b96-1c46-4f39-b8d4-e8b7c291b87c/log).
```
nox > * unit-2.7: success
nox > * unit-3.6: success
nox > * unit-3.7: success
nox > * unit-3.8: success
nox > * unit-3.9: success
nox > * cover: success
nox > * lint: success
nox > * blacken: success
nox > * docs: success
nox > * doctest: success
nox > * system-2.7: success
nox > * system-3.8: success
```

Reply:
ahaha - well in that case I'll just remove the CircleCI tests, which was my original intent here